geonode/geoserver/management/commands/sync_geonode_layers.py

```diff
@@ -41,12 +41,17 @@ def sync_geonode_layers(ignore_errors, filter, username):
         try:
             count += 1
             print 'Syncing layer %s/%s: %s' % (count, layers_count, layer.name)
-            # sync permissions in GeoFence
-            perm_spec = json.loads(_perms_info_json(layer))
-            # re-sync GeoFence security rules
-            layer.set_permissions(perm_spec)
-            # recalculate the layer statistics
-            set_attributes_from_geoserver(layer, overwrite=True)
+            if updatepermissions or not updatethumbnails:
```

Comment:
@afabiani I would make these arguments True by default and simply check is they're set, without any permutive check. I.e. if none are explicitly set by the user, ot both are set, both the actions should be performed.

Reply:
@giohappy done

Reply:
@afabiani what was something between this and the previous. Ok the defaults to true but if any of them is set the other must be False, unless they are explicitely true.